Docs/source/usage/workflows.rst

```diff
@@ -11,6 +11,7 @@ This section collects typical user workflows and best practices for WarpX.
    workflows/parallelization
    workflows/debugging
    workflows/libensemble
+   workflows/plot_timestep_duration.rst
```

Comment:
Neither timing data nor tiny profile data is saved in the plot files currently.  For the time step duration data, I assume it won't be hard to do it.  But it might not be easy to do that for tiny profile data, because currently amrex::TinyProfiler only supports printing the data at the end of the run.